source/documentation/free_radius_presentation.md

```diff
@@ -0,0 +1,94 @@
+# FreeRADIUS Notes
+
+## AAA
+
+GovWifi currently has a set of 6 RADIUS servers. These servers control network access with [AAA authentication][aaa link].
+![alt text][radius_exchange]
+
+### Authentication
+
+- Runs on port 1812.
+- Requests user details stored in the database via the [Authentication API][authentication api link].
+
+### Authorisation
+
+- We do not have different levels of access so this is a NOOP.
+
+### Accounting
+
+- Runs on Port 1813.
+- Disabled for GovWifi, network admins can monitor this on their own infrastructure.
+
+## Unlang
+
+FreeRADIUS is configured using a language called [Unlang][unlang docs link].
+
+We favor keeping all complex functionality in the backend APIs, instead of in Unlang.
+
+This way it is easier to test and change in the future.
+
+![alt text][free_radius]
```

Comment:
Will this override the alternate text defined bellow?

Might need to change to `![free_radius]`